sdk/eventhub/event-hubs/src/eventDataBatch.ts

```diff
@@ -376,7 +376,8 @@ export class EventDataBatchImpl implements EventDataBatch {
       eventData,
       options,
       entityPath,
-      host
+      host,
+      "process"
```

Comment:
typo? we might need to add "create" operation in future, but currently we don't have one for message span on producer

Reply:
Removing operation for now